package.json

```
@@ -281,8 +281,8 @@
 		"lint:lockfile": "node ./bin/validate-package-lock.js",
 		"lint:tsconfig": "node ./bin/validate-tsconfig.mjs",
 		"lint:md:docs": "wp-scripts lint-md-docs",
-		"prelint:php": "wp-env run composer \"update --no-interaction\"",
-		"lint:php": "wp-env run composer run-script lint",
+		"prelint:php": "wp-env run --env-cwd=\"wp-content/plugins/gutenberg\" cli composer \"update --no-interaction\"",
```

```suggestion
		"prelint:php": "wp-env run --env-cwd='wp-content/plugins/gutenberg' cli composer update --no-interaction",
```

I like using single quotes since it takes up less space, but that's subjective. but I do think we can remove the final quotes here now